thruster_control/src/thruster_control.cpp

```diff
@@ -105,65 +107,85 @@ ThrusterControl::ThrusterControl(ros::NodeHandle& nodeHandle)
+  {
+    diagnostic_tools::Diagnostic::WARN, ReportFault::THRUSTER_RPM_STALE
+  };
+  paramsReportsRPMCheckPeriod.abnormal_diagnostic(diagnosticRPMInfoStale);
   diagnosticsUpdater.add(publisher_reportRPM.add_check<diagnostic_tools::PeriodicMessageStatus>(
-      "rate check", diagnostic_tools::PeriodicMessageStatusParams{}
-                        .min_acceptable_period(minReportRPMRate)
-                        .max_acceptable_period(maxReportRPMRate)
-                        .abnormal_diagnostic({diagnostic_tools::Diagnostic::WARN,
-                                              health_monitor::ReportFault::THRUSTER_RPM_STALE})));
+      "rate check", paramsReportsRPMCheckPeriod));
 
   publisher_reportMotorTemp =
       diagnostic_tools::create_publisher<thruster_control::ReportMotorTemperature>(
           nodeHandle, "/thruster_control/report_motor_temperature", 1);
 
+
+  diagnostic_tools::PeriodicMessageStatusParams paramsReportsTemperatureCheckPeriod{};
+  paramsReportsTemperatureCheckPeriod.min_acceptable_period(minReportMotorTemperatureRate);
+  paramsReportsTemperatureCheckPeriod.max_acceptable_period(maxReportMotorTemperatureRate);
+  diagnostic_tools::Diagnostic diagnosticTemperatureInfoStale
+  {
+      diagnostic_tools::Diagnostic::WARN, ReportFault::THRUSTER_TEMP_STALE
+  };
+  paramsReportsTemperatureCheckPeriod.abnormal_diagnostic(diagnosticTemperatureInfoStale);
   diagnosticsUpdater.add(
       publisher_reportMotorTemp.add_check<diagnostic_tools::PeriodicMessageStatus>(
-          "rate check",
-          diagnostic_tools::PeriodicMessageStatusParams{}
-              .min_acceptable_period(minReportMotorTemperatureRate)
-              .max_acceptable_period(maxReportMotorTemperatureRate)
-              .abnormal_diagnostic({diagnostic_tools::Diagnostic::ERROR,
-                                    health_monitor::ReportFault::THRUSTER_TEMP_STALE})));
+          "rate check", paramsReportsTemperatureCheckPeriod));
 
   diagnosticsUpdater.add(
       publisher_reportMotorTemp.add_check<diagnostic_tools::MessageStagnationCheck>(
           "stagnation check",
           [motorTemperatureSteadyBand](const thruster_control::ReportMotorTemperature& a,
-                                       const thruster_control::ReportMotorTemperature& b) {
+                                       const thruster_control::ReportMotorTemperature& b)
+          {
             return std::fabs(a.motor_temp - b.motor_temp < motorTemperatureSteadyBand);
-          },
-          diagnostic_tools::MessageStagnationCheckParams{}.stagnation_diagnostic(
-              {diagnostic_tools::Diagnostic::WARN,
-               health_monitor::ReportFault::THRUSTER_TEMP_STAGNATED})));
+          }
+          , diagnostic_tools::MessageStagnationCheckParams{}
+          .stagnation_diagnostic(
+              {
+                diagnostic_tools::Diagnostic::WARN, ReportFault::THRUSTER_TEMP_STAGNATED
+              }
```

Comment:
@patapufate nit: perhaps re-arranging this temporaries like we did for the `battery_monitor` is best (for readability).

Reply:
OK